.github/workflows/test.yml

```diff
@@ -12,6 +12,11 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
+      - name: setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: '7.4'
```

Comment:
This is the  fix we want in an ideal setup where every piece of our infrastructure is up to date.  But given that our [Docker image is still stuck at 7.2](https://github.com/localgovdrupal/drupal-container/blob/596e0bc03241c240d1f7bf05ea5c53ba55a233ac/Dockerfile#L17), how about settling for 7.2 here as well?  That will at least let this test pass for now.

Reply:
I reverted mine in the alert banner to 7.2 until we have the 7.4 tests fixed.
https://github.com/localgovdrupal/localgov_alert_banner/pull/103/files

Reply:
Thanks for the pointer Andy.  Tebb, [Andy's change](https://github.com/localgovdrupal/localgov_alert_banner/pull/103/files) is what you need for the [localgov_roles PR](https://github.com/localgovdrupal/localgov/pull/168/files).

FAO @tebb